README.md

```diff
@@ -63,8 +72,18 @@ Your Answer:
 argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
-```js
-// write code here
+```
+var doSomething = function(thingToDo) {
+  thingToDo();
+};
+
+function goToSleep() {
+  if (time >= midnight) {
+    return sleep
```

Comment:
Y'alls fancy. But this would error out because `sleep`, `time`, and `midnight` are undefined. -3000 points.